cron_scripts/password_expiration_notification.pl

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long;
+use Data::Dumper;
+use JSON;
+use File::Slurp qw(read_file);
+use POSIX qw(strftime);
+use Pod::Usage qw(pod2usage);
+use Time::Piece;
+use FindBin;
+use Parallel::ForkManager;
+use LWP::UserAgent;
+
+=head1 NAME
+
+password_expiration_notification.pl - notify end users to change their password before expire
+
+=head1 SYNOPSIS
+
+  password_expiration_notification.pl [-help] [-debug] -pass_expiration_days <numExpDays> -notification_days <numNotiDays> -mail_file <mailFilePath> -sms_file <smsFilePath> -max_proc <maxNumProc>
```

Comment:
Should this:
   ```password_expiration_notification.pl [-help] [-debug] -pass_expiration_days <numExpDays> -notification_days <numNotiDays> -mail_file <mailFilePath> -sms_file <smsFilePath> -max_proc <maxNumProc>```
be this (as pass_expiration_days and notification_days seem to have default values):
   ```password_expiration_notification.pl [-help] [-debug] [-pass_expiration_days <numExpDays>] [-notification_days <numNotiDays>] -mail_file <mailFilePath> -sms_file <smsFilePath> -max_proc <maxNumProc>```

May also want to make ```--max_proc``` an optional argument and set a default value for this?